foo_uie_console/main.cpp

```diff
@@ -252,6 +266,7 @@ void ConsoleWindow::get_config(stream_writer* writer, abort_callback& abort) con
 {
     writer->write_lendian_t(current_config_version, abort);
     writer->write_lendian_t(static_cast<int32_t>(m_edge_style), abort);
+    writer->write_lendian_t(static_cast<int32_t>(m_append_newline), abort);
```

Comment:
Should be able to write out the boolean as it is (as a single byte rather than an int32_t).